examples/runner-certificates/README.md

```diff
@@ -0,0 +1,216 @@
+
+  # Mount EC2 host certs in docker so all user docker images can reference them.
+  # Each user image will need to do:
+  # cp /etc/gitlab-runner/certs/* /usr/local/share/ca-certificates/
+  # update-ca-certificates
+  # Or similar OS-dependent commands. The above are an example for Ubuntu.
+  runners_additional_volumes = ["/etc/gitlab-runner/certs/:/etc/gitlab-runner/certs:ro"]
+
+  ###############################################
+  # Registration
+  ###############################################
+
+  gitlab_runner_registration_config = {
+    registration_token = var.registration_token
+    tag_list           = "docker_runner"
+    description        = "runner docker - auto"
+    locked_to_project  = "true"
+    run_untagged       = "false"
+    maximum_timeout    = "3600"
+  }
+
+  ###############################################
+  # Network
+  ###############################################
+  vpc_id    = module.vpc.vpc_id
+  subnet_id = element(module.vpc.public_subnets, 0)
+
+}
+```
+----
+
+## Documentation
+
+----
+### Inputs
+
+| Name | Description | Type | Default | Required |
+|------|-------------|------|---------|:--------:|
+| <a name="input_aws_region"></a> [aws\_region](#input\_aws\_region) | AWS region. | `string` | `"eu-west-1"` | no |
+| <a name="input_environment"></a> [environment](#input\_environment) | A name that identifies the environment, will used as prefix and for tagging. | `string` | `"runners-docker"` | no |
+| <a name="input_gitlab_url"></a> [gitlab\_url](#input\_gitlab\_url) | URL of the gitlab instance to connect to. | `string` | `"https://gitlab.com"` | no |
+| <a name="input_registration_token"></a> [registration\_token](#input\_registration\_token) | Gitlab runner registration token | `string` | `"something"` | no |
+| <a name="input_runner_name"></a> [runner\_name](#input\_runner\_name) | Name of the runner, will be used in the runner config.toml | `string` | `"docker"` | no |
+
+----
+### Modules
+
+| Name | Source | Version |
+|------|--------|---------|
+| <a name="module_runner"></a> [runner](#module\_runner) | ../../ | n/a |
+| <a name="module_vpc"></a> [vpc](#module\_vpc) | terraform-aws-modules/vpc/aws | 2.70 |
+
+----
+### Outputs
+
+No outputs.
+
+----
+### Requirements
+
+| Name | Version |
+|------|---------|
+| <a name="requirement_terraform"></a> [terraform](#requirement\_terraform) | >= 1 |
+| <a name="requirement_aws"></a> [aws](#requirement\_aws) | ~> 4.7 |
+| <a name="requirement_local"></a> [local](#requirement\_local) | ~> 2 |
+| <a name="requirement_null"></a> [null](#requirement\_null) | ~> 3.0 |
+| <a name="requirement_random"></a> [random](#requirement\_random) | ~> 3.0 |
+| <a name="requirement_tls"></a> [tls](#requirement\_tls) | ~> 3 |
+
+----
+### Resources
+
+| Name | Type |
+|------|------|
+| [aws_availability_zones.available](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/availability_zones) | data source |
+
+----
+<!-- END_TF_DOCS -->
```

Comment:
suggestion: remove everything between the `TF_DOCS` tags. It's automatically generated as soon as the branch is merged into `develop`.